Automattic/jetpack
# PHPCS: Publicize

Fixes https://github.com/Automattic/jetpack/issues/21968.

This is the first step in abstracting Publicize while also conveniently fixing PHPCS violations at the same time.

#### Changes proposed in this Pull Request:

* Fix PHPCS violations in Publicize module.

#### Jetpack product discussion



n/a

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* Make sure CI is still happy.
* Try using Publicize features: add a new connection, publish a post, make sure it's posted to the social media connection.
